### ISSUE TITLE
Ignore definitely not working on the Developers folder.

### ISSUE THREAD

**Issue by elamhut** [SRombauts/UE4PlasticPlugin#85]
I reported this issue to the Plastic SCM team that my ignore isn't working and they said it is an issue with the plugin. so here's my report, my video and my ignore.conf:

 Our developers folder is completely ignored as you can see below. (It was at some point commited but has since been deleted from the repository)

<img width="395" alt="01" src="https://user-images.githubusercontent.com/36922414/119526063-6dfb7d00-bd55-11eb-9313-860c1ef64432.png">

This is our ignore file

<img width="209" alt="02" src="https://user-images.githubusercontent.com/36922414/119526078-718f0400-bd55-11eb-99a0-668e94fff915.png">

But whenever I add something to it (yes we use Plastic's Unreal Plugin but the team commits through Plastic/Gluon):

<img width="1158" alt="03" src="https://user-images.githubusercontent.com/36922414/119526097-73f15e00-bd55-11eb-8ddf-04e44e93d97e.png">

Plastic is all like "Nah brah, let's commit dat"

<img width="360" alt="04" src="https://user-images.githubusercontent.com/36922414/119526108-7784e500-bd55-11eb-832e-b6b7dc214586.png">

Our lock rules even state that "PLEASE DON'T TRY TO LOCK FILES IN THIS FOLDER"

<img width="376" alt="05" src="https://user-images.githubusercontent.com/36922414/119526115-7a7fd580-bd55-11eb-9c69-75399198f05a.png">

In the video you can see that my folders are Private/Ignored and even then the Plugin adds any files that I create in these folders to the source control. I'd be expecting the ignore.conf to be respected but it's clearly not, right now.

https://www.youtube.com/watch?v=YvAPeRA7cs4

(Had to change extension to txt so github would accept it)
[ignore_conf.txt](https://github.com/SRombauts/UE4PlasticPlugin/files/6540566/ignore_conf.txt)

**Comment by REGmolin** [comment]
I just submitted a ticket for Plastic regarding this. The ignore file is not respected when using the plugin.

This is the repro case I submitted

Repro case (Using UE4 DefaultEditor.ini as an example)

1. Make sure Config/DefaultEditor.ini isn't checked into the repository or in the "Added and private items".
2. Add the file (DefaultEditor.ini) to ignore.conf
3. Start UE4
4. Make changes in the Preferences ("Edit / Editor Preferences..." in the Menu). Edit something that gets stored in the DefaultEditor.ini
5. See if the file is checked out in the Plastic client
